app/src/main/java/com/eveningoutpost/dexdrip/utilitymodels/UpdateActivity.java

```diff
@@ -80,6 +80,9 @@ public class UpdateActivity extends BaseAppCompatActivity {
     private static String CHECKSUM = "";
 
     public static void checkForAnUpdate(final Context context) {
+        StackTraceElement[] stacktrace = Thread.currentThread().getStackTrace();
```

Comment:
This isn't a good way to achieve this when you can simply add an alternate method signature. I will do this and you can use that to add the UI element.